bin/gdc-client

```diff
@@ -76,6 +76,7 @@ if __name__ == '__main__':
         dest='command',
         help='for more information, specify -h after a command',
     )
+    subparsers.required = True
```

Comment:
Can we add this as a method parameter instead?

https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_subparsers